themes/digital.gov/src/scss/new/shortcodes/_note.scss

```diff
@@ -1,139 +1,146 @@
-    }
+  // For notes within list items
+  li .note {
+    @include u-padding-y("05");
+    @include u-padding-x(1);
   }
-
-  .highlight {
-    @include u-bg("orange-warm-10");
-  }
-}
-
-// Activities  — color settings
-.note.alert {
-  @include u-bg("red-5");
-  @include u-text("red-60v");
-  @include u-border-left("05", "solid", "red-40v");
-
-  a {
-    @include u-text("red-60v");
-    @include u-border-bottom("1px", "solid", "red-60v");
-
-    &:hover {
-      @include u-text("red-80v");
+  // Activities  — color settings
+  &--activity {
+    background-color: color("orange-warm-5");
+    border-left-color: color("orange-warm-30");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("orange-warm-60");
+
+    a {
+      border-bottom-color: color("orange-warm-60");
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+      color: color("orange-warm-60");
+
+      &:hover {
+        color: color("orange-warm-80");
+      }
     }
-  }
 
-  .highlight {
-    @include u-bg("red-10");
+    .highlight {
+      background-color: color("orange-warm-10");
+    }
   }
-}
-
-// Comment — color settings
-.note.comment {
-  @include u-bg("blue-cool-10");
-  @include u-text("blue-cool-70");
-  @include u-border-left("05", "solid", "blue-cool-50");
+  // Alert styles
+  &--alert {
+    background-color: color("red-5");
+    border-left-color: color("red-40v");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("red-60v");
+
+    a {
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+      color: color("red-60v");
+
+      &:hover {
+        color: color("red-80v");
+      }
+    }
 
-  h4 {
-    i {
-      @include u-text("blue-cool-60");
+    .highlight {
+      background-color: color("red-10");
     }
   }
+  // Comment — color settings
+  &--comment {
+    background-color: color("blue-cool-10");
+    border-left-color: color("blue-cool-50");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("blue-cool-80");
+
+    a {
+      border-bottom-style: solid;
+      border-bottom-width: 1px;
+
+      &:hover {
+        color: color("blue-cool-80");
+      }
+    }
 
-  a {
-    @include u-text("blue-cool-70");
-    @include u-border-bottom("1px", "solid", "blue-cool-50");
+    .highlight {
+      background-color: color("white");
+    }
 
-    &:hover {
-      @include u-text("blue-cool-80");
+    blockquote {
+      font-family: family("serif");
+      font-size: size("ui", "lg");
     }
   }
-
-  .highlight {
-    @include u-bg("white");
-    @include u-bg("yellow-5");
+  &--disclaimer {
+    background-color: color("gray-cool-3");
+    border-left-color: color("gray-cool-20");
+    border-left-style: solid;
+    border-left-width: 5px;
+    color: color("gray-warm-90");
   }
+}
 
-  blockquote {
-    @include u-font("serif", "lg");
-  }
+.dg-note__heading {
+  font-family: family("sans");
+  font-size: 20px;
+  font-weight: 200;
```

Comment:
Please use token. 

_If this was supposed to be light it should be `300`._

Reply:
Updated to use tokens